docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Titanium_SDK_Overview.md

```diff
@@ -5,34 +5,17 @@ weight: '30'
 
 # Titanium SDK Overview
 
-Welcome to the Appcelerator Platform! The Appcelerator Platform helps you develop cross-platform mobile applications using the Titanium SDK, Alloy, Appcelerator Studio and the Titanium CLI, then lets you manage the entire lifecycle of the application with debugging, testing, deploying, crash monitoring and analytic data collection.
+Welcome to the Appcelerator Platform! The Appcelerator Platform helps you develop cross-platform mobile applications using the Titanium SDK, Alloy and the Titanium CLI, then lets you manage the entire lifecycle of the application with debugging, testing, deploying, crash monitoring and analytic data collection.
```

Comment:
Shouldn't `Appcelerator Platform` be also replaced? I'm not really sure if `TiDev Platform` is the best expression. @cb1kenobi what do you think?

Reply:
There a re plenty of occurrences of `Appcelerator Platform` all over the docs